redis.py

```diff
@@ -21,9 +29,8 @@ def main():
             key, value = x.split(':')
             redis_stats[key] = value
 
-    hostname = socket.gethostname().replace('.', '_')
-    timestamp = str(int(time.time()))
-    template = 'servers.' + hostname + '.software.redis.{1} {2} ' + timestamp
+    timestamp = str(int(time()))
+    template = args.prefix + '.{1} {2} ' + timestamp
```

Comment:
Is `timestamp` neede somewhere else. If not, you can just use `int(time())` inline.